edge-modules/api-proxy-module/src/monitors/certs_monitor.rs

```diff
@@ -16,12 +16,13 @@ const PROXY_SERVER_CERT_PATH: &str = "/app/server.crt";
 const PROXY_SERVER_PRIVATE_KEY_PATH: &str = "/app/private_key_server.pem";
 
 const PROXY_SERVER_VALIDITY_DAYS: i64 = 90;
+const CERTIFICATE_POLL_INTERVAL: tokio::time::Duration = tokio::time::Duration::from_secs(1);
```

Comment:
is tokio:: needed here?

Reply:
It is necessary here or inside the code because we need a duration. I can move it inside the code if it is ugly here

Reply:
no, just a nit - I mean can you do time::Duration instead of tokio::time::Duration